.eslintrc.js

```diff
@@ -1,27 +1,13 @@
 module.exports = {
-
   root: true,
-
-  parserOptions: {
-    'ecmaVersion': 2017,
+  extends: ['@metamask/eslint-config', '@metamask/eslint-config-mocha'],
```

Comment:
We should only extend the Mocha config in the test file override.

Reply:
Whoops! Good catch. I had actually thought of this and checked the README for this config to see what was recommended, and the example there doesn't use an override. In that draft there was no override yet so I didn't bother. But now that it's here anyway, there's really no reason not to migrate it.